src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx

```diff
@@ -289,6 +290,10 @@ function getLogoSrcs(changes: TransactionChanges): string[] {
 }
 
 function parseUniswapXOrder({ details, chain, timestamp }: OrderActivity): Activity | undefined {
+  // We currently only have a polling mechanism for locally-sent pending orders, so we hide remote pending orders since they won't update upon completion
+  // TODO(WEB-2487): Add polling mechanism for remote orders to allow displaying remote pending orders
+  if (details.orderStatus === SwapOrderStatus.Open) return undefined
```

Comment:
Does this mean that if you send an order and refresh the page it will disappear from your mini portfolio activity?

Reply:
No, redux persists